### PR TITLE
refactor(service): change service interface return type

### PR DESCRIPTION
**Pull request by KaoImin**
**What type of PR is this?**
refactor

**What this PR does / why we need it**:
* Change service interface return type from `Result<T, ServiceResponse<()>>` to `ServiceResponse<T>`.
* Add a `try_service_response` macro for handling service response.

e.g.
```rust
/// Expand code
/// let balance = {
///     if resp.is_error() {
///         return ServiceResponse::from_error(resp.code, resp.error_message);     
///     }
///     resp.succeed_data
/// };
let resp = asset.balance_(ctx, payload);
let balance = try_service_response!(resp);
```

The corresponding doc PR is https://github.com/nervosnetwork/muta-docs/pull/36/files

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by KaoImin** [comment]
ping @yejiayu

File: benchmark/governance/mod.rs [hunk 1]
```diff
@@ -8,6 +8,7 @@ use bytes::Bytes;
 use derive_more::{Display, From};
 
 use binding_macro::{genesis, hook_after, service, tx_hook_after, tx_hook_before};
+use protocol::parse_resp;
```
Comment (yejiayu) [inline benchmark/governance/mod.rs:11]: The name `parse_resp` doesn’t make sense

**Comment by KaoImin** [comment]
This PR is ready to be merged @yejiayu.

**Pull request merged**